juan-aristizabal-bedoya/tarea11-routing/routing/src/components/Navbar.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import "./Navbar.css";
+
+const Navbar = () => {
+  return (
+    <article className="navbar-container">
+
+      <p className="center-text">Routing homework</p>
+    <ul className="navbar-ul">
+      <li className="navbar-li">
```

Comment:
@Juan-Aristizabal , list should be build dynamically from an Array using map function